### ISSUE TITLE
Undefined offset: 0

### ISSUE THREAD

**Issue by markbratanov** [ericbae/XTA2#4]
I can reproduce this on my server and the XTA2 demo hosted by ericbae.

Create an account using google, enter user name and email and I receive:

A PHP Error was encountered
Severity: Notice
Message: Undefined offset: 0
Filename: controllers/auth_other.php
Line Number: 295

A PHP Error was encountered
Severity: Notice
Message: Trying to get property of non-object
Filename: controllers/auth_other.php
Line Number: 295

A PHP Error was encountered
Severity: Warning
Message: Cannot modify header information - headers already sent by (output started at /home/ecatv/public_html/system/core/Exceptions.php:170)
Filename: helpers/url_helper.php
Line Number: 541

I'm assuming these errors could be disregarded when CI is in production mode, but I still think this is an error to be looked at.

**Comment by emko** [comment]
i get this error only when i type a username the exists. it never tells me that its in use, it does tell me if email is in use. anyone have any idea how to fix this?

**Comment by emko** [comment]
nvm figured it out. the username input validation needs a call back to check if user exists.

**Comment by markbratanov** [comment]
emko, could you commit those changes?

**Comment by markbratanov** [comment]
Nevermind.

**Comment by emko** [comment]
sorry im not very good with php so thats why i just discibed the problem and how i think its fixed. Did it fix the probem for you?

**Comment by markbratanov** [comment]
I'm going to take a look at it more closely now, since you've given me a lead. Thanks. When I figure it out I'll let you know!

**Comment by emko** [comment]
i made a pull request to fix this

**Comment by markbratanov** [comment]
I just took a look at your pull request... yea, it looks awesome. Great job!

Do you have it working in your instance?

**Comment by emko** [comment]
yes i tested it and it will show user name already exist error instead  of trying to go on and giving the Undefined offset: 0
